Update description for filter_parameter_logging

jess opened the pull request:
Looking at the list and description, it's not very clear that the filter_parameters are partial matches. For example I was wondering why password wasn't on the list and if I needed to add it or if there was a default list somewhere and this initializer was just adding to the default list.

jess committed:
Update description for filter_parameter_logging

robzolkos commented on railties/lib/rails/generators/rails/app/templates/config/initializers/filter_parameter_logging.rb.tt:
```diff
@@ -1,8 +1,8 @@
-# sensitive information. See the ActiveSupport::ParameterFilter documentation for supported
-# notations and behaviors.
+# Configure parameters to be partially matched (e.g. passw matches password) and filtered from the log file. 
+# Use this to limit dissemination ofsensitive information. 
```
```suggestion
# Use this to limit dissemination of sensitive information. 
```

robzolkos reviewed (commented):
Typo

jess committed:
Update railties/lib/rails/generators/rails/app/templates/config/initializers/filter_parameter_logging.rb.tt

Thanks Rob!!. I did all this in the github ui and was a little out of sorts with the wrapping.

Co-authored-by: Rob Zolkos <rob@zolkos.com>

jess commented:
@robzolkos it appears the build failed. From what I can tell, it doesn't have anything do to do with the change we made. Any idea how to trigger it to run again? May it just needs another commit??

robzolkos commented:
@jess yeah rebase main and force push - should trigger a build.
